test/src/bgw/params.c

```diff
@@ -112,6 +112,7 @@ params_open_wrapper()
 			StartTransactionCommand();
 #endif
 		seg = dsm_attach(handle);
+		Assert(seg != NULL);
```

Comment:
dsm_attach can return NULL, might be safer to turn the Assert into proper check otherwise next function call with dsm will segfault